yaml/com/ibm/Control/Host/PcieLink.interface.yaml

```diff
@@ -0,0 +1,10 @@
+description: >
+    Pcie Link interface is used to reset the link
+    to true or false.
+
+properties:
+    - name: linkReset
+      type: boolean
+      default: false
+      description: >
+          Set this to reset the Pcie Link.
```

Comment:
Can you add a bit more here? Does the service that monitors for this move it back false? True means a reset of the link should happen ?

Reply:
done